internal/config/aws.go

```diff
@@ -54,7 +54,7 @@ func (d *aws) loadConfig() {
 			panic(errors.Wrap(err, "failed to figure out"))
 		}
 
-		file, err := os.Create("/root/.aws/credentials")
+		file, err := os.Create("/Users/mike/root/.aws/credentials")
```

Comment:
Code run in a docker container, don't change this line. For local run, you can comment it but don't commit it to external repo